### PR TITLE
lang: Improving Russian translation

### PR DESCRIPTION
**Pull request by AHOHNMYC**
## Description
Improving Russian translation and adding missing phrases.

### CODE REVIEW

**Review by misteroneill — APPROVED** [review]
Changes look good to me, but I don't speak Russian.

@roman-bc-dev Would you be able to take a look at this one?

File: lang/ru.json [hunk 1]
```diff
@@ -72,14 +74,16 @@
...
+  "Casual": "Казуальный",
+  "Script": "Рукописный",
+  "Small Caps": "Капитель",
   "Reset": "Сбросить",
-  "restore all settings to the default values": "сбросить все найстройки по умолчанию",
+  "restore all settings to the default values": "сбросить все настройки на значения по умолчанию",
   "Done": "Готово",
   "Caption Settings Dialog": "Диалог настроек подписи",
-  "Beginning of dialog window. Escape will cancel and close the window.": "Начало диалоговго окна. Кнопка Escape закроет или отменит окно",
+  "Beginning of dialog window. Escape will cancel and close the window.": "Начало диалогового окна. Нажмите Escape для отмены и закрытия окна",
```
Comment (roman-bc-dev) [inline lang/ru.json:84]: Yikes. Good change here!

File: lang/ru.json [hunk 2]
```diff
@@ -72,14 +74,16 @@
   "Monospace Sans-Serif": "Моноширинный без засечек",
   "Proportional Serif": "Пропорциональный с засечками",
   "Monospace Serif": "Моноширинный с засечками",
-  "Casual": "Случайный",
-  "Script": "Письменный",
-  "Small Caps": "Малые прописные",
+  "Casual": "Казуальный",
+  "Script": "Рукописный",
+  "Small Caps": "Капитель",
```
Comment (roman-bc-dev) [inline lang/ru.json:79]: This is a bit of a gap where my knowledge of typography doesn't square well with my knowledge of Russian. Small caps isn't exactly just capital letters, it's replacing lowercase letters with diminutive versions of a type-face's capitol characters. Does the word `Капитель` reflect that?

Comment (AHOHNMYC) [reply lang/ru.json:79]: Yes! You may check this: [wiki:Small caps](https://en.wikipedia.org/wiki/Small_caps) has link to Russian article named [Капитель](https://ru.wikipedia.org/wiki/Капитель_(шрифт)). Another independent proofs are in [Ushakov's dictionary](https://ushakovdictionary.ru/word.php?wordid=22845), [Paratype's glossary](https://info.paratype.ru/glossary/к#Капитель) and [Bureau Gorbunov's advice](https://bureau.ru/bb/soviet/20101206/).

**Review by roman-bc-dev — APPROVED** [review]

**Pull request merged**